### DISCUSSION TITLE
Feature idea: curl --count

### DISCUSSION THREAD

**Discussion by jidanni** [curl/curl#10515] [Ideas]
Let's say one is getting 456 files. As one stares at the screen one
cannot tell how far along we are. Are we already at the 300th file?

Feature idea:
- `--count`
- - In progress messages curl will put a count next to each item retrieved.

Sure, one instead could just kludge up:
`$ curl ... 2>&1 | perl -ple 'print ++$k if /Xferd/'`

**Comment by abhint** [comment]
Can you show any examples or explain them further?

**Comment by jidanni** [comment]
Sure, a config file hundreds of
```
url = "https://commo
output = "Dealey_Pla

url = "https://commo
output = "Dealey_Pla

url = "https://commo
output = "Anapra_New

url = "https://commo
output = "San_Simon_

url = "https://commo
output = "Modesto_CA
```
long. While the job is running all we see are

```
  % Total    % Rece

100  1256  100  125
  % Total    % Rece

100  1256  100  125
  % Total    % Rece

100  1256  100  125
```
with no idea if we are on the 30th or 300rd file.
